src/Visitor/MutationsCollectorVisitor.php

```diff
@@ -183,8 +169,12 @@ private function getOuterMostArrayNode(Node $node): Node
     /**
      * @return array|int[]
      */
-    private function getNodeRange(Node $node): array
+    private function getNodeRange(Node $node, bool $isOnFunctionSignature): array
```

Comment:
Side note (it doesn't need to be addressed now): I think `bool $onFunctionSignature` is better as the `is` or `has` prefix are more suited for their corresponding getters